java/compaction/compaction-job-creation/src/test/java/sleeper/compaction/job/creation/AwsCompactionJobDispatcherIT.java

```diff
@@ -234,4 +238,14 @@ private List<CompactionJobDispatchRequest> receiveDeadLetters() {
                 .map(new CompactionJobDispatchRequestSerDe()::fromJson).toList();
     }
 
+    private void assertThatFilesAreInBucket() {
+        Set<String> contents = listObjectKeys(instanceProperties.get(DATA_BUCKET));
+        assertThat(contents).isNotEmpty();
+    }
+
+    private void assertThatFilesAreNotInBucket() {
+        Set<String> contents = listObjectKeys(instanceProperties.get(DATA_BUCKET));
+        assertThat(contents).isEmpty();
+    }
```

Comment:
It might be worth making it clear that what matters is whether the batch file is deleted. Something like assertThatBatchFileWasDeleted, assertThatBatchFileWasNotDeleted?